### PR TITLE
Hindi and Bangla translations for alerts

### PR DESCRIPTION
**Pull request by prem-k-r**
## Description
hindi translations for alerts and color palette

## Checklist
- [x] I have read and followed the [Contributing Guidelines](../CONTRIBUTING.md).
- [x] I have added necessary tests.
- [x] The project builds and runs without issues.

### CODE REVIEW

**Review by itz-rj-here — APPROVED** [review]
The Bangla Translations are correct great work :)

**Comment by itz-rj-here** [comment]
Still some mistakes found cuz of the new english translation
w8 i will do the rest

File: languages.js [hunk 1]
```diff
@@ -806,8 +823,24 @@ const translations = {
 
         // Wallpaper and alerts
         "uploadWallpaperText": "ওয়ালপেপার দিন", //"ওয়ালপেপার আপলোড করুন"
-        "backupText": "ব্যাকআপ",
-        "restoreText": "পুনরুদ্ধার",
+        "backupText": "ব্যাকআপ করুন",
+        "restoreText": "পুনরুদ্ধার করুন",
+        "rangColor": "রঙ বাছাই",
```
Comment (prem-k-r) [inline languages.js:828]: রঙ নির্বাচন করুন
this one is still within the box, so this can be replaced

Comment (itz-rj-here) [reply languages.js:828]: np i already did

**Pull request merged**